CHANGELOG.md

```diff
@@ -11,17 +11,34 @@
     - Removes the ability to directly set `state` by making it `private(set)`. This prevents users from bypassing reducers and middleware. All mutation of the state must occur through the normal `Action` & `Reducer` methods.
     - This deprecates the usage of `ReSwift-Recorder`. Changes may be made to that library in the future in order to support this change.
 
+**Other**
+
+# 4.1.1
+
+*Released: 03/21/2019*
+
+- Fix 4.1.0 regression with `automaticallySkipsRepeats` when selecting Equatable state in Non-Equatable root state (#399) - @djtech42
+
+# 4.1.0
+
+*Released: 03/21/2019*
+
+**API Changes:**
```

Comment:
You _could_ drop the `:` here for consistency with `**Other**`, but that's not really important, is it? :)